src/resource-archive/index.test.ts

```diff
@@ -95,29 +111,105 @@ describe('new', () => {
     expect(mockWarn).toBeCalledWith(`Global timeout of 1ms reached`);
   });
 
-  // eslint-disable-next-line jest/expect-expect
   it('gathers basic resources used by the page', async () => {
-    const complete = await createResourceArchive(page);
+    const complete = await createResourceArchive({ page });
 
     await page.goto(baseUrl);
 
     const archive = await complete();
 
-    expectArchiveContains(archive, ['/img.png', '/style.css'], pathToResponseInfo, baseUrl);
+    expect(archive).toEqual({
+      'http://localhost:13337/style.css': {
+        statusCode: 200,
+        statusText: 'OK',
+        body: Buffer.from(styleCss),
+        contentType: 'text/css; charset=utf-8',
+      },
+      'http://localhost:13337/img.png': {
+        statusCode: 200,
+        statusText: 'OK',
+        body: Buffer.from(imgPng, 'base64'),
+        contentType: undefined,
+      },
+    });
```

Comment:
I inlined these expected-cases since I was having a hard time telling what exactly `createResourceArchive` was supposed to output.